src/main/kotlin/no/nav/api/utbetalinger/UtbetalingerClient.kt

```diff
@@ -85,14 +87,15 @@ class UtbetalingerClient(
     
     suspend fun hentUtbetalinger(fnr: String, fra: LocalDate, til: LocalDate): List<Utbetaling> = externalServiceCall {
         val request: UtbetaldataRequest = lagUtbetaldataRequest(fnr, fra, til)
-        try {
-            client.post("$utbetalingerUrl/v1/hent-utbetalingsinformasjon/intern") {
-                contentType(ContentType.Application.Json)
-                body = request
-            }
-        } catch (ex: Exception) {
-            throw WebStatusException(
-                message = ex.message ?: "Henting av utbetalinger for bruker med fnr $fnr mellom $fra og $til feilet.",
```

Comment:
Ville egentlig beholde denne exception-meldingen 🤔 Hva er beste måte å både håndtere ulike statuskoder og kunne få ut en exception fra deres api?

Reply:
Mulig vi kan bruke `response.readText()` for å hente ut body'en til som er returneret.
Evt må ting wrappes i en try-catch (evt `runCatching`) om man ønsker å fange alle feil som kan skje